string timestamp
bool startbag
bool startrecord
bool startwave1
bool startwave2
bool shutdown
